lib/final_destination.rb

```diff
@@ -16,7 +16,7 @@ def self.clear_https_cache!(domain)
 
   def self.cache_https_domain(domain)
     key = redis_https_key(domain)
-    Discourse.redis.without_namespace.setex(key, "1", 1.day.to_i).present?
+    Discourse.redis.without_namespace.setex(key, 1.day.to_i, "1").present?
```

Comment:
Wow, we were calling it with wrong arguments?! 😲

Reply:
The upshot was we were setting a non-zero value for 1 second, rather than setting a non-zero value for 1 day. Meaning we have have been doing a few more `GET`s than necessary, although with most sites defaulting to https this probably wasn't a huge issue.